src/hotspot/cpu/x86/stubGenerator_x86_64_tanh.cpp

```diff
@@ -0,0 +1,502 @@
+    420909223,  3163117379, 4112506593, 1071816349, 2947355221, 1014371048,
+    1972484976, 1071813050, 675290301,  3161640050, 3790955393, 1071809759,
+    2352942462, 3163180090, 874372905,  1071806478, 100263788,  1015940732,
+    1709341917, 1071803205, 2571168217, 1014152499, 1897844341, 1071799941,
+    1254300460, 1015275938, 1337108031, 1071796686, 3203724452, 1014677845,
+    4219606026, 1071793439, 2434574742, 1014681548, 1853186616, 1071790202,
+    3066496371, 1015656574, 2725843665, 1071786973, 1433917087, 1014838523,
+    2440944790, 1071783753, 2492769774, 1014147454, 897099801,  1071780542,
+    754756297,  1015241005, 2288159958, 1071777339, 2169144469, 1014876021,
+    2218315341, 1071774145, 2694295388, 3163288868, 586995997,  1071770960,
+    41662348,   3162627992, 1588871207, 1071767783, 143439582,  3162963416,
+    828946858,  1071764615, 10642492,   1015939438, 2502433899, 1071761455,
+    2148595913, 1015023991, 2214878420, 1071758304, 892270087,  3163116422,
+    4162030108, 1071755161, 2763428480, 1015529349, 3949972341, 1071752027,
+    2068408548, 1014913868, 1480023343, 1071748902, 2247196168, 1015327453,
+    948735466,  1071745785, 3516338028, 3162574883, 2257959872, 1071742676,
+    3802946148, 1012964927, 1014845819, 1071739576, 3117910646, 3161559105,
+    1416741826, 1071736484, 2196380210, 1011413563, 3366293073, 1071733400,
+    3119426314, 1014120554, 2471440686, 1071730325, 968836267,  3162214888,
+    2930322912, 1071727258, 2599499422, 3162714047, 351405227,  1071724200,
+    3125337328, 3159822479, 3228316108, 1071721149, 3010241991, 3158422804,
+    2875075254, 1071718107, 4144233330, 3163333716, 3490863953, 1071715073,
+    960797498,  3162948880, 685187902,  1071712048, 378731989,  1014843115,
+    2952712987, 1071709030, 3293494651, 3160120301, 1608493509, 1071706021,
+    3159622171, 3162807737, 852742562,  1071703020, 667253586,  1009793559,
+    590962156,  1071700027, 3829346666, 3163275597, 728909815,  1071697042,
+    383930225,  1015029468, 1172597893, 1071694065, 114433263,  1015347593,
+    1828292879, 1071691096, 1255956747, 1015588398, 2602514713, 1071688135,
+    2268929336, 1014354284, 3402036099, 1071685182, 405889334,  1015105656,
+    4133881824, 1071682237, 2148155345, 3162931299, 410360776,  1071679301,
+    1269990655, 1011975870, 728934454,  1071676372, 1413842688, 1014178612,
+    702412510,  1071673451, 3803266087, 3162280415, 238821257,  1071670538,
+    1469694871, 3162884987, 3541402996, 1071667632, 2759177317, 1014854626,
+    1928746161, 1071664735, 983617676,  1014285177, 3899555717, 1071661845,
+    427280750,  3162546972, 772914124,  1071658964, 4004372762, 1012230161,
+    1048019041, 1071656090, 1398474845, 3160510595, 339411585,  1071653224,
+    264588982,  3161636657, 2851812149, 1071650365, 2595802551, 1015767337,
+    4200250559, 1071647514, 2808127345, 3161781938
+};
+
+#define __ _masm->
+
+address StubGenerator::generate_libmTanh() {
```

Comment:
Please add the link to original source references from where algorithm is ported / disassembled.